synapse/lib/tags.py

```diff
@@ -43,6 +43,8 @@ def getTufoSubs(tufo, tag):
     prop = '*|%s|%s' % (form,tag)
     if tufo[1].get(prop):
         props.append(prop)
+    else:
+        return props
 
     pref = '*|%s|%s.' % (form,tag)
```

Comment:
```pref = prop + '.'``` is quicker than doing the string % creation a second time.  See https://gist.github.com/vEpiphyte/500ac7ec1ecc8fb019f55c4e7fb82561